shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
That's fine... In that case, do you think you could reset the `clean-out-ie9-css-hacks` branch to the CSS commit and re-push it? You should be able to do that with `git reset --hard d34cabe130ee0a64141030bec44cad2600ef45e6` and then `git push origin clean-out-ie9-css-hacks --force`?